v2/blockstore/ro_blockstore.go

```diff
@@ -0,0 +1,190 @@
+package blockstore
```

Comment:
this package is called blockstore, so I think blockstore/readonly.go is slightly better than blockstore/ro_blockstore.go